vms/proposervm/block.go

```diff
@@ -38,6 +38,8 @@ var (
 	errProposerMismatch         = errors.New("proposer mismatch")
 	errProposersNotActivated    = errors.New("proposers haven't been activated yet")
 	errPChainHeightTooLow       = errors.New("block P-chain height is too low")
+	errInvalidVRFSignature      = errors.New("invalid VRF signature")
+	errVRFSignaturePresents     = errors.New("unexpected VRF signature presents")
```

Comment:
wording nit. I think just changing `presents` to `present` would be ok too.
```suggestion
	errUnexpectedVRFSignature     = errors.New("unexpected VRF signature present")
```

Reply:
done